jar-persistence/src/main/java/com/sixsq/slipstream/persistence/VirtualMachine.java

```diff
@@ -20,8 +21,63 @@ public String toJson() {
     public static class ServiceOfferRef{
         public final String href;
 
+        @SerializedName("resource:vcpu")
+        public final Integer resourceVcpu;
+
+        @SerializedName("resource:ram")
+        public final Float resourceRam;
+
+        @SerializedName("resource:disk")
+        public final Float resourceDisk;
+
+        @SerializedName("resource:instanceType")
+        public final String resourceInstanceType;
+
+        @SerializedName("price:currency")
+        public final String priceCurrency;
+
+        @SerializedName("price:unitCost")
+        public final Float priceUnitCost;
+
+        @SerializedName("price:billingPeriodCode")
+        public final String priceBillingPeriodCode;
+
+        @SerializedName("price:freeUnits")
+        public final Float priceFreeUnits;
+
+        @SerializedName("price:unitCode")
+        public final String priceUnitCode;
+
         public ServiceOfferRef(String href) {
```

Comment:
Could be simpler with 
public ServiceOfferRef(String href) {
            this(href, null, null, null, null, null, null, null, null, null);
        }